src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -53,15 +56,26 @@ const ImagePreviewContainer = styled.View`
   marginBottom: 10
 `
 
+const PDFPreviewContainer = styled.View`
+  marginBottom: 10
+`
+
 interface Props extends RelayProps {
   senderName: string
   artworkPreview?: JSX.Element
-  imagePreview?: JSX.Element
 }
 
 export class Message extends React.Component<Props, any> {
   render() {
     const date = moment(this.props.message.created_at).fromNow(true)
```

Comment:
It does, you can request `created_at(format: "...")`. It doesn't support the `fronNow` relative format we seem to be using in Messaging though. So that would have to be added in Metaphysics, make sure all the date fields are correctly typed as `date` there, and then we can drop this out of the code and either get Metaphysics to just return it that way, or request it via params in our query.